markdown/extensions/tables.py

```diff
@@ -85,7 +90,49 @@ def _split_row(self, row, border):
                 row = row[1:]
             if row.endswith('|'):
                 row = row[:-1]
-        return row.split('|')
+        return self._split(row, '|')
+
+    def _split(self, row, marker):
+        """ split a row of text with some code into a list of cells. """
+        if self._row_has_unpaired_backticks(row):
+            # fallback on old behaviour
+            return row.split(marker)
+        backtick_pattern = BacktickPattern('^' + BACKTICK_RE) # modify the backtick pattern to only match at the beginning of the search string
+        elements = []
+        current = ''
+        i = 0
+        while i < len(row):
+            letter = row[i]
+            if letter == marker:
+                if current != '' or len(elements) == 0:
+                    # Don't append empty string unless it is the first element
+                    # The border is already removed when we get the row, then the line is strip()'d 
+                    # If the first element is a marker, then we have an empty first cell
+                    elements.append(current)
+                current = ''
+                escaped = False
+            else:
+                match = backtick_pattern.getCompiledRegExp().match(row[i:])
+                if not match:
+                    current += letter
+                else:
+                    groups = match.groups()
+                    delim = groups[1]  # the code block delimeter (ie 1 or more backticks)
+                    row_contents = groups[2]  # the text contained inside the code block
+                    i = row.index(groups[3])  # jump pointer to the beginning of the rest of the text
```

Comment:
Any reason why you're not using `match.end(group)` here?

Every thing else looks good to me.